bot/seasons/evergreen/speedrun.py

```diff
@@ -0,0 +1,29 @@
+import json
+import logging
+from pathlib import Path
+from random import choice
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Speedrun(commands.Cog):
+    """Commands about the video game speedrunning community."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="speedrun")
+    async def get_speedrun(self, ctx):
+        """Sends a link to a video of a random speedrun."""
+        with open(Path('bot/resources/evergreen/speedrun_links.json')) as file:
```

Comment:
Sorry for not catching this with my first review. It would be better to load the JSON file only once when the cog is loaded rather than loading it every time the command is called.

Reply:
Good point, would it make sense to do this on init, creating a self.data to hold the value? Or should I be trying to do this in the setup function?

Reply:
You can either do it in `__init__` or do it at the top of the module and store it as a module-level constant. Both ways have been used for other features on seasonalbot. It's up to you.

Reply:
Module level is preferred

Reply:
Whoops, @sco1 just saw your comment after I pushed the commit. I'll get it moved to the module level. I'm also going to change its name from `data` to `LINKS` so it's a bit more descriptive.